docs/source/pickling_cuml_models.ipynb

```diff
@@ -0,0 +1,781 @@
+{
```

Comment:
<p>Do I remember that we want to clear output from notebooks before merging in order to run them in CI? If so, don't forget to clear before we merge.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuml/pull/3154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>That's a great point! I think that mean I need to add the dask-ml and matplotlib dependencies somewhere as well. </p>